community/modules/scheduler/htcondor-service-accounts/outputs.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "access_point_service_account_email" {
+  description = "HTCondor Access Point Service Account (e-mail format)"
+  value       = module.access_point_service_account.service_account_email
+  depends_on = [
+    module.access_point_service_account
+  ]
+}
+
+output "central_manager_service_account_email" {
+  description = "HTCondor Central Manager Service Account (e-mail format)"
+  value       = module.central_manager_service_account.service_account_email
+  depends_on = [
+    module.central_manager_service_account
+  ]
+}
+
+output "execute_point_service_account_email" {
+  description = "HTCondor Execute Point Service Account (e-mail format)"
+  value       = module.execute_point_service_account.service_account_email
+  depends_on = [
```

Comment:
nit. Please add some comment about `depends_on`? 
Looks like this problem ( I presume "email is available before all resources in the module are created" ) should be solved in `community/modules/project/service-account` , instead of boiling it up to the consumer, WDYT?

Reply:
I agree it looks like it already is solved in community/modules/project/service-account. I'll remove the dependency and add a comment.